lxc/console.go

```diff
@@ -339,6 +339,12 @@ func (c *cmdConsole) vga(d lxd.InstanceServer, name string) error {
 		cmd.Stderr = os.Stderr
 		cmd.Start()
 
+		// Handle the command exitting.
```

Comment:
Typo

Reply:
I'll put the typo fix in my master branch so it goes up with whatever next bugfix I send :)